dashboard/src/components/OpenGraphTags/OpenGraphTags.tsx

```diff
@@ -0,0 +1,29 @@
+import type { JSX } from 'react';
+
+export const OpenGraphTags = ({
+  title,
+  url,
+  description,
+  imageUrl = 'https://dashboard.kernelci.org/kernelci-logo-card.png',
+  type = 'website',
+}: {
+  title: string;
+  url: string;
+  description: string;
+  imageUrl?: string;
+  type?: string;
+}): JSX.Element => {
+  return (
+    <>
+      <meta property="og:title" content={title} />
+      <meta property="og:url" content={url} />
+      <meta property="og:description" content={description} />
+      <meta property="og:image" content={imageUrl} />
+      <meta property="og:type" content={type} />
+      <meta property="twitter:title" content={title} />
+      <meta property="twitter:site" content={url} />
+      <meta property="twitter:description" content={description} />,
+      <meta property="twitter:image" content={imageUrl} />
```

Comment:
Here's why we were having a comma at the top of every page
```suggestion
      <meta property="og:image" content={imageUrl} />
      <meta property="og:type" content={type} />
      <meta property="twitter:title" content={title} />
      <meta property="twitter:site" content={url} />
      <meta property="twitter:description" content={description} />
      <meta property="twitter:image" content={imageUrl} />
```

Reply:
pesky comma